FXyz-Samples/src/main/java/org/fxyz3d/samples/importers/ImportSample.java

```diff
@@ -0,0 +1,171 @@
+import javafx.scene.Group;
+import javafx.scene.Node;
+import javafx.scene.Parent;
+import javafx.scene.shape.CullFace;
+import javafx.scene.shape.DrawMode;
+import javafx.scene.shape.MeshView;
+import javafx.scene.transform.Rotate;
+import org.fxyz3d.controls.ControlCategory;
+import org.fxyz3d.controls.NumberSliderControl;
+import org.fxyz3d.controls.factory.ControlFactory;
+import org.fxyz3d.importers.Importer3D;
+import org.fxyz3d.importers.Model3D;
+import org.fxyz3d.samples.shapes.ShapeBaseSample;
+import org.fxyz3d.shapes.polygon.PolygonMeshView;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ *
+ * @author JosePereda
+ */
+public abstract class ImportSample extends ShapeBaseSample<Group> {
+
+    protected final IntegerProperty subdivision = new SimpleIntegerProperty(this, "Subdivision Level", 0);
+    protected final ObjectProperty<URL> url = new SimpleObjectProperty<>(this, "URL");
+    protected final ObjectProperty<Timeline> timeline = new SimpleObjectProperty<>(this, "timeline");
+    protected final ObjectProperty<Node> content = new SimpleObjectProperty<>(this, "content");
+    protected final BooleanProperty asPolygonMesh = new SimpleBooleanProperty(this, "asPolygonMesh", true);
+    protected final BooleanProperty yUp = new SimpleBooleanProperty(this, "yUp", true);
+    
+    @Override
+    protected void addMeshAndListeners() {
+        drawMode.addListener((obs, b, b1) -> {
+            if (model != null) {
+                setDrawMode(model, b1);
+            }
+        });
+        culling.addListener((obs, b, b1) -> {
+            if (model != null) {
+                setCullFace(model, b1);
+            }
+        });
+        subdivision.addListener((obs, ov, nv) -> setSubdivisionLevel(model, nv.intValue()));
+        url.addListener((obs, ov, nv) -> {
+            try {
+                Model3D model3D = asPolygonMesh.get() ?
+                        Importer3D.loadAsPoly(url.get()) : Importer3D.load(url.get());
+                model.getChildren().setAll(model3D.getRoot().getChildren());
+                model3D.getTimeline().ifPresentOrElse(t -> {
+                    model.sceneProperty().addListener(new InvalidationListener() {
+                        @Override
+                        public void invalidated(Observable observable) {
+                            if (model.getScene() == null) {
+                                model.sceneProperty().removeListener(this);
+                                timeline.set(null);
+                            }
+                        }
+                    });
+                    timeline.set(t);
+                }, () -> timeline.set(null));
+                initModel();
+            } catch (IOException e) {
+                Logger.getLogger(ImportSample.class.getName()).log(Level.SEVERE,null, e);
+            }
+        });
+        yUp.addListener((obs, ov, nv) -> {
```

Comment:
Can we bind the rotate angle value? (didn't check syntax)

```
Rotate rt = new Rotate(Rotate.X_AXIS);
rt.angleProperty().bind(Bindings.when(yUp).then(180).otherwise(0));
```

Reply:
Doing 
```  
model.setRotationAxis(Rotate.X_AXIS);
model.rotateProperty().bind(Bindings.when(yUp).then(180).otherwise(0));
````
kind of works, but it rotates the model 180º over its center and now, in case of the duke king, the model has its 0 in the top of the hat. Whereas the transform rotates over 0, and flips the model 180º, so the 0 still is at duke's feet.

Reply:
I wasn't suggesting to bind the mode's angle, but to bind the `Rotate` transform's angle. Would that work? 

```
Rotate rt = new Rotate(Rotate.X_AXIS);
rt.angleProperty().bind(Bindings.when(yUp).then(180).otherwise(0));
model.getTransforms().setAll(rt);
```

Reply:
Yes, that works